src/platform/site-wide/va-footer/components/Footer.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import links from '../../../static-data/footer-links.json';
+import { isWideScreen } from '../../../utilities/accessibility/index';
+import CrisisPanel from './CrisisPanel';
+import DesktopLinks from './DesktopLinks';
+import MobileLinks from './MobileLinks';
+import { createLinkGroups } from '../helpers';
+
+export default class Footer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.linkObj = createLinkGroups(links);
+    this.state = {
+      isMobile: !isWideScreen(),
+    };
+  }
+  componentDidMount() {
+    // TODO: debounce
+    window.addEventListener(
+      'resize',
+      () => {
+        if (this.state.isMobile !== !isWideScreen()) {
+          this.setState({
+            isMobile: !isWideScreen(),
+          });
+        }
+      },
+      false,
+    );
+    if (this.props.handleFooterDidMount) {
+      this.props.handleFooterDidMount();
+    }
+  }
+  render() {
+    return (
+      <div>
+        <div className="footer-inner">
+          <DesktopLinks visible={!this.state.isMobile} links={this.linkObj} />
+          <MobileLinks visible={this.state.isMobile} links={this.linkObj} />
```

Comment:
If it's possible to wrap the Desktop/Mobile components in a class that shows only for those screen sizes, we might be able to get away with removing `isMobile` from state.

Reply:
We need to dynamically set `aria-hidden` either way, so I figured it wasn't worth doing at the moment.

Reply:
Ah yeah good call